packages/vite/src/node/baseEnvironment.ts

```diff
@@ -14,6 +14,7 @@ export function getDefaultResolvedEnvironmentOptions(
   config: ResolvedConfig,
 ): ResolvedEnvironmentOptions {
   return {
+    name: '$',
```

Comment:
I think the `name` should be passed from the argument.

Reply:
The name here is a dummy value for defaults. I don't think it would help to pass it from the two places where this function is called. Ideally, it shouldn't be there, but I don't want to make it nullable.